src/vma/util/agent.cpp

```diff
@@ -77,12 +77,14 @@ agent* g_p_agent = NULL;
 
 agent::agent() :
 		m_state(AGENT_CLOSED), m_sock_fd(-1), m_pid_fd(-1),
-		m_msg_num(512), m_msg_grow(16)
+		m_msg_num(512), m_msg_grow(16),
+		m_inactive_treshold(10), m_alive_treshold(10)
```

Comment:
1. Please change all hard coded numbers to be under #define
2. Currently inactive threshold and alive thresholds cannot be configured. Please suggest how to be able to modify them (can be a separate PR)